Dockerfile

```diff
@@ -71,23 +83,18 @@ RUN mix do deps.get, deps.compile, assets.deploy, sentry.package_source_code, re
 # Stage: runtime
 ########################################################################
 
-FROM alpine:3.20.3@sha256:beefdbd8a1da6d2915566fde36db9db0b524eb737fc57cd1367effd16dc0d06d as runtime
+FROM ${RUNTIME_IMAGE} AS runtime
 
 ENV APP_HOME=/opt/app
 WORKDIR $APP_HOME
 
-RUN apk --no-cache --update upgrade \
-    && apk add --no-cache bash ca-certificates libstdc++ openssl ncurses-libs \
-    && update-ca-certificates --fresh
-
-SHELL ["/bin/bash", "-c"]
+RUN apt-get update -y \
+    && apt-get install -y libstdc++6 openssl libncurses5 locales ca-certificates wget \
+    && apt-get clean && rm -f /var/lib/apt/lists/*_*
 
-COPY --from=builder /etc/bashrc.d /etc/bashrc.d
-COPY --from=builder /etc/bash.bashrc /etc/bash.bashrc
 COPY --from=builder /opt/scripts /opt/scripts
-COPY .docker/root/.bashrc /root/
 
-RUN chown -R nobody:nobody /opt
+RUN chown -R nobody /opt
```

Comment:
This needs to be done in the two places below (`COPY`, `USER`), too - I think.

Reply:
👍